deisctl/units/deis-swarm-manager.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=deis-cache
+
+[Service]
+EnvironmentFile=/etc/environment
+TimeoutStartSec=20m
+ExecStartPre=/bin/sh -c "docker history smothiki/swarm >/dev/null || docker pull smothiki/swarm"
```

Comment:
We need to figure out etcd master to bootstrap swarm nodes and manager . The custom image has deis wrapper for swarm which starts swarm nodes and swarm managers and finds ETCD master. Custom image is builds swarm binary from my fork where I have soft affinity implementations and little experiments with health checks